docs/usage-as-end-user.md

```diff
@@ -264,6 +276,11 @@ spec:
       volume:
         size: 50Gi
         type: cloud_efficiency
+        # NOTE: This comment is for the case when encrypted of an existing shoot is updated from false to true. 
+        # It will cause affected nodes rolling updated. It is must to MAINTAIN the shoot. 
```

Comment:
```suggestion
        # It will cause affected nodes rolling updated. It must MAINTAIN the shoot. 
```
```suggestion
        # It will cause affected nodes rolling updated. It is must to MAINTAIN the shoot. 
```